### PR TITLE
RSWW-12 added room options scraping

### PR DESCRIPTION
**Pull request by CzajaLukasz**

### CODE REVIEW

File: logic/tours/scrapping_single_tour.py [hunk 1]
```diff
@@ -173,6 +175,70 @@ def get_tour_photos(driver: WebDriver) -> List[str]:
         return []
 
 
+def get_room_options(driver: WebDriver):
```
Comment (Danzigerrr) [inline logic/tours/scrapping_single_tour.py:188]: please add a short doc-string describing this method

**Comment by Danzigerrr** [comment]
I was testing it on the link below and it didn't work:

https://www.itaka.pl/wczasy/zjednoczone-emiraty-arabskie/abu-dhabi/hotel-khalidiya-palace-rayhaan-by-rotana,AAEAUH1WKO.html?id=CgVJdGFrYRIEVklUWBoDUExOIgpBQUVBVUgxV0tPKAQ6BEtMMjBCBgiAkeizBkoGCICd%252FbMGUAJiBQoDS1JLagUKA0FVSHIICgZEUDMwMDh6BQoDQVVIggEFCgNLUkuKAQgKBkRQMzAwOJIBBgiAkeizBpoBBgiAnf2zBqIBDAoKUk1TRDAwMDBCMKoBAwoBQQ%253D%253D&participants[0][adults]=2

Here is the error message:

```
Traceback (most recent call last):
  File "C:\Informatyka\8 SEMESTR\RSWW\logic\tours\scrapping_single_tour.py", line 379, in <module>
    test_tour = scrape_single_tour(driver, test_tour)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Informatyka\8 SEMESTR\RSWW\logic\tours\scrapping_single_tour.py", line 367, in scrape_single_tour
    rooms = get_room_options(driver)
            ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Informatyka\8 SEMESTR\RSWW\logic\tours\scrapping_single_tour.py", line 180, in get_room_options
    person_button = driver.find_element(By.XPATH, "//*[@class='styles_c__Muqrv styles_wrapper__1Wod9']/button[2]")
                    
  File "C:\Informatyka\8 SEMESTR\RSWW\venv\Lib\site-packages\selenium\webdriver\remote\webdriver.py", line 741, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Informatyka\8 SEMESTR\RSWW\venv\Lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "C:\Informatyka\8 SEMESTR\RSWW\venv\Lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@class='styles_c__Muqrv styles_wrapper__1Wod9']/button[2]"}
  (Session info: chrome=122.0.6261.112); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

**Comment by Danzigerrr** [comment]
I think it is a good idea to add this piece of code to _initalize_scraper.py_ just after the variable _driver_ is initialized:
`driver.maximize_window()`

**Comment by Danzigerrr** [comment]
I was testing [this link](https://www.itaka.pl/wczasy/hiszpania/bilbao/hotel-vincci-consulado-de-bilbao,1015174.html?id=CgVJdGFrYRIEVklURBoDUExOIgcxMDE1MTc0KAM6BEtMMjBCBgiAt%252FivBkoGCICgiLAGUAGSAQYIgLf4rwaaAQYIgKCIsAaiAQUKA0RCTKoBAwoBVQ%253D%253D&participants%5B0%5D%5Badults%5D=2)
and encoutered an error:

```
Traceback (most recent call last):
  File "C:\Informatyka\8 SEMESTR\RSWW\logic\tours\scrapping_single_tour.py", line 383, in <module>
    test_tour = scrape_single_tour(driver, test_tour)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Informatyka\8 SEMESTR\RSWW\logic\tours\scrapping_single_tour.py", line 369, in scrape_single_tour
    rooms = get_room_options(driver)
            ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Informatyka\8 SEMESTR\RSWW\logic\tours\scrapping_single_tour.py", line 206, in get_room_options
    room_button = room.find_element(By.XPATH, ".//ul/button")
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Informatyka\8 SEMESTR\RSWW\venv\Lib\site-packages\selenium\webdriver\remote\webelement.py", line 417, in find_element
    return self._execute(Command.FIND_CHILD_ELEMENT, {"using": by, "value": value})["value"]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Informatyka\8 SEMESTR\RSWW\venv\Lib\site-packages\selenium\webdriver\remote\webelement.py", line 395, in _execute
    return self._parent.execute(command, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Informatyka\8 SEMESTR\RSWW\venv\Lib\site-packages\selenium\webdriver\remote\webdriver.py", line 347, in execute
    self.error_handler.check_response(response)
  File "C:\Informatyka\8 SEMESTR\RSWW\venv\Lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":".//ul/button"}
```

**Comment by CzajaLukasz** [comment]
Fixed. Now it should work

**Review by Danzigerrr — APPROVED** [review]
Well done @CzajaLukasz! Works nice! 😄 👨‍💻

**Pull request merged**